modules/data/src/main/java/io/fluo/webindex/data/fluo/DomainMap.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2015 Fluo authors (see AUTHORS)
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.fluo.webindex.data.fluo;
+
+import java.util.Iterator;
+
+import com.google.common.base.Optional;
+import io.fluo.api.client.TransactionBase;
+import io.fluo.api.config.FluoConfiguration;
+import io.fluo.api.observer.Observer.Context;
+import io.fluo.recipes.export.ExportQueue;
+import io.fluo.recipes.map.CollisionFreeMap;
+import io.fluo.recipes.map.CollisionFreeMap.Options;
+import io.fluo.recipes.map.Combiner;
+import io.fluo.recipes.map.Update;
+import io.fluo.recipes.map.UpdateObserver;
+import io.fluo.webindex.data.FluoApp;
+import io.fluo.webindex.data.recipes.Transmutable;
+
+public class DomainMap {
```

Comment:
It's cool how you put both the `Combiner` and `UpdateObserver` implementations in the same class.  It makes the code easier to read. You could update the CFM documentation to follow this pattern.